beps/0002-dynamic-frontend-plugins/dependency-sharing-schema.json

```
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "type": "object",
+  "properties": {},
+  "definitions": {
+    "semverString": {
+      "type": "string",
+      "pattern": "^(\\^|\\~)?(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
+    }
+  },
+  "patternProperties": {
+    "^(?:@([^/]+?)[/])?([^/]+?)$": {
```

```suggestion
    "^(@[^/]+/)?[^/]+$": {
```

Same thing in the end right? Just a bit less going on :p